.github/workflows/linters.yml

```diff
@@ -6,14 +6,14 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  eslint:
```

Comment:
I'll respectfully undo these changes; I know it's not a big deal but I think keeping the full name of the lint would keep things clear.